alpharaw/wrappers/alphatims_wrapper.py

```diff
@@ -45,25 +45,26 @@ def import_raw(self, burker_d_folder: str):
 class AlphaTimsWrapper(TimsTOF):
     """Create a AlphaTims object that contains
     all data in-memory (or memory mapping).
-
-    Parameters
-    ----------
-    msdata : MSData_Base
-        The AlphaRaw data object.
-
-    dda : bool
-        If DDA, precursor indices will be equal to scan numbers.
-        If not DDA (i.e. DIA), precursor indices will be equal to the
-        scan number within a DIA cycle.
-
-    slice_as_dataframe : bool
-        If True, slicing returns a pd.DataFrame by default.
-        If False, slicing provides a np.int64[:] with raw indices.
-        This value can also be modified after creation.
-        Default is True.
     """
 
     def __init__(self, msdata: MSData_Base, dda: bool, slice_as_dataframe: bool = True):
+        """
+        Parameters
+        ----------
+        msdata : MSData_Base
+            The AlphaRaw data object.
+
+        dda : bool
+            If DDA, precursor indices will be equal to scan numbers.
+            If not DDA (i.e. DIA), precursor indices will be equal to the
+            scan number within a DIA cycle.
+
+        slice_as_dataframe : bool
+            If True, slicing returns a pd.DataFrame by default.
+            If False, slicing provides a np.int64[:] with raw indices.
+            This value can also be modified after creation.
```

Comment:
"This value can also be modified after creation." -> what does that mean?

Reply:
This module will be deprecated, as alphadia has its own implementation already, so I did not work on it any more.

Reply:
See https://github.com/MannLabs/alphadia/blob/main/alphadia/data/alpharaw.py#L251

Reply:
> wont FIX:)

Lets MB keep it ..

Reply:
This is required by the super class AlphaTims, it must be set here to enable alphatims functionalities.

Reply:
I will move this to the attribute definition. It means this attr can be modified on-the-fly.

Reply:
Moved this to attribute definition.